src/test/scala/org/broadinstitute/dsde/firecloud/service/WorkspaceServiceSpec.scala

```diff
@@ -40,10 +40,14 @@ class WorkspaceServiceSpec extends BaseServiceSpec with BeforeAndAfterEach {
         status
       }
 
-      val tsvReturnString = List(
+      val tsvReturnStringOld = List(
         List("workspace:f", "d", "b", "a", "e", "c").mkString("\t"),
         List("[\"v6\",999,true]", "escape quo\\\"te", 1.23, "true", "v1", "").mkString("\t")).mkString("\n")
 
+      val tsvReturnString = List(
+        List("workspace:e", "d", "b", "c", "a", "f").mkString("\t"),
+        List("v1", "escape quo\\\"te", 1.23, "", "true", "[\"v6\",999,true]").mkString("\t")).mkString("\n")
```

Comment:
the column order returned by this code changed, not sure why- maybe some underlying implementation to collections?

Reply:
@davidangb can you think of any problematic side effects of this?

Reply:
looking at this now - the ordering changes are concerning on the surface but might be totally harmless.